README.md

```diff
@@ -1,2 +1,19 @@
 # HW01
-Edit the readme
+## Student name: Johnathon Hall
+I am a rising 5th year PhD student in the **CMB** graduate program.
+A few of my scientific interests are:
+
+* noncoding RNA
+* enhancers
+* chromatin
+* gene regulation
+* plant biology
+* cardiology
+
+
+![Here is a photo of me^](/Users/rnahe/OneDrive/Pictures/john.jpg)
+
+Here is my lab's website:
+http://www.ruthenlab.org/
+
+Looking forward to learning R with all of you!
```

Comment:
Hey Jonathon, I think I'm in your group. I'm trying to catch up on the material as I'm a bit busy with my current job (I was not expecting a the Quant Bio to be a summer online course LOL)
I hope I'm not screwing up the HW XP